README.md

```diff
@@ -33,12 +170,13 @@ To verify that it works run
 pip install -r requirements.txt
 python -m pytest -s ./test
```

Comment:
This actually won't work that way anymore since those tests would take hours to run. I'd suggest
```
python -m pytest -s ./test/unit
```
which should take about 10 minutes. I still find that way too long, so maybe we need to tag a few tests to run for this sanity check, so that it takes no more than 10 seconds or so?